### PR TITLE
Displaying login error properly

### LINKED ISSUE #663: Get username from jr_kayttaja table and use that when saving
FE PR: https://github.com/HSLdevcom/jore-map-ui/pull/903
BE PR: https://gitlab.hsl.fi/jore/jore-map-backend/merge_requests/67

When signing in and user has write access, backend tries to get user with hsl-id account's email address
* if email is not found, redirect user to login page and show error to the user "Kirjautuminen epäonnistui, sähköpostiosoitteen omaavaa käyttäjätunnusta ei löytynyt joukkoliikennerekisterin tietokannasta. Ota yhteyttä käyttäjätunnusten hallinnoijaan."

### PR DESCRIPTION
**Pull request by Divinare**
Closes #663 

BE PR: https://gitlab.hsl.fi/jore/jore-map-backend/merge_requests/67

- textContainer fails to display "MUOKATTU" timestamp, edited description of card #895 to fix it
- showing login failure information if username was not found
   - also logging out when login fails

### CODE REVIEW

File: src/util/text.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import textCodeList from '~/codeLists/textCodeList';
```
Comment (kronken) [inline src/util/text.ts]: The naming needs to be changed. 
A file named text.ts, with a function named text doesn't tell that much

File: src/util/text.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+ * @param {keyValueMap} { key: value } - key is the same as ${key} in textCodeListKey, ${key} is replaced with value
+ **/
+const text = (textCodeListKey: string, keyValueMap?: Object) => {
+    let s = textCodeList[textCodeListKey];
+    const re = /\$\{(\w+)\}/g; // ${...}
```
Comment (kronken) [inline src/util/text.ts]: const regexRule

File: src/util/text.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import textCodeList from '~/codeLists/textCodeList';
+
+/**
+ * @param {keyValueMap} { key: value } - key is the same as ${key} in textCodeListKey, ${key} is replaced with value
+ **/
+const text = (textCodeListKey: string, keyValueMap?: Object) => {
+    let s = textCodeList[textCodeListKey];
```
Comment (kronken) [inline src/util/text.ts]: let message

Comment (Divinare) [reply src/util/text.ts]: I put lineString there now

File: src/util/text.ts (hunk 1 above)
Comment (Divinare) [reply src/util/text.ts]: CodeListHelper & getText

**Comment by Divinare** [comment]
Review fixes ^

File: src/util/text.ts (hunk 3 above)
Comment (kronken) [reply src/util/text.ts]: sure, everything is better than 's' :D

**Pull request merged**